### PR TITLE
[issue 32328] Update Dataviz disabled color

### PR DESCRIPTION
**Pull request by shawn-grant**
Updated `DataVizGradientPalette.disabled` color to E6E6E6

## Related Issue(s)
#32328

### CODE REVIEW

**Review by AtishayMsft — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-41e2919e-7823-4579-a692-b0432e5669cb.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-41e2919e-7823-4579-a692-b0432e5669cb.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32340&commitId=a0d8a242f510ba26c01b6660db2a6ce64c2bfc9a&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (AtishayMsft) [reply change/@fluentui-react-charting-41e2919e-7823-4579-a692-b0432e5669cb.json:1]: No noticable difference observed in the 2 screenshots. Resolving this comment

**Pull request merged**